.github/workflows/deploy.yml

```diff
@@ -24,6 +24,9 @@ jobs:
       - name: Build
         run: yarn build
 
+      - name: Copy index.html to 404.html
+        run: cp ./dist/index.html ./dist/404.html
```

Comment:
for the reviewer -- when GH pages serves static files, if they don't find anything at the target url they will serve 404.html.

they will only serve static file if there is a repo named that with gh pages. like we are at `github.io/turbo-app` and they will show that but if someone goes to `github.io/turbo-app/gift` they show 404 by default.

By adding this dup 404.html we basically create the app there as well as at index. So GH pages serves our 404.html at subpaths which is actually our app

Reply:
we have a workaround for this in the arns-react app - https://github.com/ar-io/arns-react/blob/develop/404.html

Reply:
I was testing a similar script last night but this seems to solve the problem in a simpler way